### ISSUE TITLE
Videos don't play 

### ISSUE THREAD

**Issue by VagrantZTV** [yuliskov/SmartTube#3727]

**Comment by gate1975mlm** [comment]
Did you update to latest version 22.44?

22.44 is working great for me on my Shield TV Pro and Sony A80J TV.

All 4K videos that I tried work and no more 403 errors! :)

I sure hope it stays this way for a long time!!!

**Comment by necrovamp** [comment]
I was on the previous version and it wasn't showing my subscriptions etc, updated to 22.44 now it does show my subscriptions but all videos have 403 here

**Comment by koffeedude** [comment]
Update SmartTube beta 22.47 solved all problems for me. I got 4K playback and no more 403 errors. I am using the newest Amazon fire cube.

**Comment by ytsejam1138** [comment]
SmartTube Beta 22.47 hasn't fixed the 403 error for me. In fact, it's gotten worse. I used to be able to start a video, 403 error would pop up, if I then selected a different video it would play and then I would be able to play the original video without any issues. With Beta 22.47 no video will play through without the 403 error. 
I've uninstalled, reinstalled, cleared data, cleared cache, logged out, logged in. I haven't found any workaround that works.

**Comment by mossbros** [comment]
I'm also still experiencing this error on the latest beta build, video will play for around a minute, then 403.
Installed on a brand new 4k fire stick including the voice search hook.

**Comment by Shibombisu** [comment]
> I'm also still experiencing this error on the latest beta build, video will play for around a minute, then 403. Installed on a brand new 4k fire stick including the voice search hook.

Experiencing same issue, regardless of video format or buffer, around 1 minute mark, 403 error. Beta 22.47 on Nvidia Shield.

**Comment by OrPgDeatruct** [comment]
> > I'm also still experiencing this error on the latest beta build, video will play for around a minute, then 403. Installed on a brand new 4k fire stick including the voice search hook.
> 
> Experiencing same issue, regardless of video format or buffer, around 1 minute mark, 403 error. Beta 22.47 on Nvidia Shield.

Exactly the same for me also on an Nvidia Shield 2019 I cannot play anything no matter the resolution or video I just get the error

**Comment by drewbitt** [comment]
Yes, 22.47 beta seems borked for me after about 1min into a video with a 403. Revert back to 22.44 stable.

**Comment by zanzibar707** [comment]
22.48 issue is still around

**Comment by ytsejam1138** [comment]
22.48 BETA is behaving the same for me on the 2017 Nvidia Shield.  However, on Stable 22.44 it plays more videos without the 403 error.  If it does have a 403 error on a video, just waiting will usually correct itself and resume the video automatically.

**Comment by Patraputt** [comment]
Updated to 22.49, everything was working fine for just 3 minutes and then ghost playback started again! Videos "playing" with no video and no audio!

**Comment by krasseboehmer** [comment]
Seems like it's only the ghost playback (black screen, no sound) for logged in accounts. When I use the default non log in user it plays back.

**Comment by VagrantZTV** [comment]
Videos are playing but erro 403 appears frequently

**Comment by Patraputt** [comment]
Updated to 22.51, Ghost playback is still persistent. Videos "playing" with no video and no audio!

**Comment by Patraputt** [comment]
Updated to 22.55, Ghost playback is still persistent. Videos "playing" with no video and no audio!

**Comment by OrPgDeatruct** [comment]
> Updated to 22.55, Ghost playback is still persistent. Videos "playing" with no video and no audio!

Thank you for the update I'll keep using beta 22.41

**Comment by gate1975mlm** [comment]
> Updated to 22.55, Ghost playback is still persistent. Videos "playing" with no video and no audio!

Can you please share a video that gives you this issue?

I want to see if I have the same issue because so far every video I play is working 100%. with 22.55 and the past few versions.

**Comment by Patraputt** [comment]
https://github.com/user-attachments/assets/2a4296cc-9e43-4112-b43d-4a40335256cc

"Ghost playback"

**Comment by Patraputt** [comment]
Updated to 22.56, Ghost playback is still persistent. Videos "playing" with no video and no audio!

**Comment by krasseboehmer** [comment]
The current update to 22.65 fixed it for me

**Comment by Pantalaim0n** [comment]
Can confirm the 22.65 update fixed the problem here

**Comment by Patraputt** [comment]
Everyone, please update to 22.65. It'll work like a charm!

**Comment by gate1975mlm** [comment]
> Everyone, please update to 22.65. It'll work like a charm!

22.65 great so far!

**Comment by ytsejam1138** [comment]
22.65 BETA and STABLE working perfectly now.

**Comment by Chealer** [comment]
@VagrantZTV : Does this issue persist? If so:

- Which version do you use?
- How do you reproduce?
- How does this differ from issue #2175?
